README.md

```diff
@@ -0,0 +1,28 @@
+
+# How to contribute
+
+If you want to contribute to a project and make it better, your help is very welcome.
+
+1. Fork the repository https://github.com/portotestersmeetup/PTM_PAGE
+2. Check the open issues that are available to the community and let us know you will tackle it
+3. When you finish, open up the pull request and we will review it as soon as possible
+4. If everything is fine, it will be merged and you will see it on the PTM website soon
+5. Later, your name will be featured in the Contributors section
```

Comment:
Shouldn't the owner of the PR add the name to the List?

Reply:
Well played

Reply:
done